shared/src/commonMain/kotlin/com/intive/picover/shared/party/di/PartyModule.kt

```diff
@@ -0,0 +1,8 @@
+package com.intive.picover.shared.party.di
+
+import com.intive.picover.parties.viewmodel.PartyDetailsViewModel
+import org.koin.dsl.module
+
+internal val partyModule = module {
```

Comment:
Have you considered using koin annotations, with compile time safety turned on? Looks promising 😄  https://insert-koin.io/docs/reference/koin-annotations/start#compile-safety---check-your-koin-config-at-compile-time-since-130

Reply:
Wow, good to know, thank you 😄  
You saved a lot of my time for future fixes of `NoBeanDefFoundException` 😄